### PR TITLE
9 store data with sequelize db

### PR DESCRIPTION
**Pull request by AnthonyRo1**
Added a backend directory to manage migrations, models, and db. 
Created migration and model for tournaments. 

installed sequelize-cli to interact with sequelize

### CODE REVIEW

File: backend/db/migrations/20230615033637-create-tournament.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      // to do: 
+      // publish_location (Channel ID)
+      // organizer_id (User.id) if user === Organizer 
+      // to be continued 
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: literal('CURRENT_TIMESTAMP')
+      },
+      updatedAt: {
```
Comment (acasaro) [inline backend/db/migrations/20230615033637-create-tournament.js:33]: Yes!👍🏼

**Review by acasaro — APPROVED** [review]
Love how organized you are. I've never used sequelize before. You clearly have. I'm glad you're handling this Nice job!

**Review by acasaro — COMMENTED** [review]
Does this mean we can import db into any file and run this logic? 

`const db = require(./backend/models`

`const { Tournament } = db`
``` javascript
const tournamentName = interaction.options.getString('name');

const tournamentDescription = interaction.options.getString('description');

try {
// equivalent to: INSERT INTO tournaments (name, description, username) values (?, ?, ?);

const tournament = await Tournament.create({
   name: tournamentName,
   description: tournamentDescription,

```

**Comment by AnthonyRo1** [comment]
> Does this mean we can import db into any file and run this logic?
> 
> `const db = require(./backend/models`
> 
> `const { Tournament } = db`
> 
> ```js
> const tournamentName = interaction.options.getString('name');
> 
> const tournamentDescription = interaction.options.getString('description');
> 
> try {
> // equivalent to: INSERT INTO tournaments (name, description, username) values (?, ?, ?);
> 
> const tournament = await Tournament.create({
>    name: tournamentName,
>    description: tournamentDescription,
> ```

Correct!

**Pull request merged**